### PR TITLE
Made compatible with Phalcon 2.0.x (View Engines)

### PR DESCRIPTION
**Pull request by tmihalik**

### CODE REVIEW

File: Library/Phalcon/Mvc/View/Engine/Twig.php [hunk 1]
```diff
@@ -20,12 +21,12 @@ class Twig extends Engine implements EngineInterface
...
-     * @param \Phalcon\DiInterface       $di
-     * @param array                      $options
-     * @param array                      $userFunctions
+     * @param ViewBaseInterface $view
+     * @param DiInterface       $di
+     * @param array             $options
+     * @param array             $userFunctions
      */
-    public function __construct($view, DiInterface $di = null, $options = array(), $userFunctions = array())
+    public function __construct(ViewBaseInterface $view, DiInterface $di = null, $options = array(), $userFunctions = array())
```
Comment (sergeyklay) [inline Library/Phalcon/Mvc/View/Engine/Twig.php]: Line exceeds 120 characters; contains 126 characters.
Can u do something with this?

**Comment by endeveit** [comment]
I think we should wait next Phalcon release where Phalcon\Mvc\ViewBaseInterface will be released

**Comment by andresgutierrez** [comment]
@endeveit ViewBaseInterface is in master https://github.com/phalcon/cphalcon/blob/master/phalcon/mvc/viewbaseinterface.zep

**Comment by sergeyklay** [comment]
Thanks

**Comment by endeveit** [comment]
@andresgutierrez yes you're right I'm little confused in branches.

**Pull request merged**